aten/src/ATen/native/MetaTensor.cpp

```diff
@@ -53,5 +53,27 @@ Tensor empty_strided_meta(
       size, stride, dtype_opt, layout_opt, device_opt, pin_memory_opt);
 }
 
+Tensor empty_strided_symint_meta(
+  SymIntArrayRef size,
+  SymIntArrayRef stride,
+  c10::optional<ScalarType> dtype_opt,
+  c10::optional<Layout> layout_opt,
+  c10::optional<Device> device_opt,
+  c10::optional<bool> pin_memory_opt
+) {
+  return at::detail::empty_strided_symint_meta(
+      size, stride, dtype_opt, layout_opt, device_opt, pin_memory_opt);
```

Comment:
??

Reply:
This is mostly cargo-culted from how `empty_symint_meta()` is implemented. It looks like `at::native::empty_symint_meta` is responsible for checking if the sizes are actually symbolic, and shunting to either the non-symbolic kernel (`at::native::empty_meta`), or the symbolic kernel (`at::detail::empty_symint_meta`)

Reply:
No what I mean is, you have a bunch of dead code below the return

Reply:
... oh welp haha